packages/app/src/App.tsx

```diff
@@ -15,30 +15,30 @@
  */
 
 import {
-  createApp,
   AlertDisplay,
-  OAuthRequestDialog,
-  SignInPage,
+  createApp,
   createRouteRef,
   FlatRoutes,
+  OAuthRequestDialog,
+  SignInPage,
 } from '@backstage/core';
-import React from 'react';
-import Root from './components/Root';
-import * as plugins from './plugins';
-import { apis } from './apis';
-import { hot } from 'react-hot-loader/root';
-import { providers } from './identityProviders';
 import { Router as CatalogRouter } from '@backstage/plugin-catalog';
-import { Router as DocsRouter } from '@backstage/plugin-techdocs';
+import { Router as ImportComponentRouter } from '@backstage/plugin-catalog-import';
+import { ExplorePage } from '@backstage/plugin-explore';
 import { Router as GraphiQLRouter } from '@backstage/plugin-graphiql';
-import { Router as TechRadarRouter } from '@backstage/plugin-tech-radar';
 import { Router as LighthouseRouter } from '@backstage/plugin-lighthouse';
 import { Router as RegisterComponentRouter } from '@backstage/plugin-register-component';
+import { Router as TechRadarRouter } from '@backstage/plugin-tech-radar';
+import { Router as DocsRouter } from '@backstage/plugin-techdocs';
 import { Router as SettingsRouter } from '@backstage/plugin-user-settings';
-import { Router as ImportComponentRouter } from '@backstage/plugin-catalog-import';
-import { Route, Navigate } from 'react-router';
-
+import React from 'react';
+import { hot } from 'react-hot-loader/root';
+import { Navigate, Route } from 'react-router';
+import { apis } from './apis';
 import { EntityPage } from './components/catalog/EntityPage';
+import Root from './components/Root';
+import { providers } from './identityProviders';
+import * as plugins from './plugins';
```

Comment:
I like this grouping of imports and I try to do it too. Do you have an extension which can do this in your editor, or was this a manual attempt?

Reply:
I don't think it's an extension, I run "Organize Imports" followed by "Format Document" (Prettier) in VSCode

Reply:
`shift + option + O` gang unite, i do this compulsively

Reply:
(but it does lead to messy diffs ;))

Reply:
First diff messy, much cleaner afterwords. I'm in.